src/Microsoft.AspNetCore.Authentication.Facebook/project.json

```diff
@@ -26,9 +26,6 @@
   "frameworks": {
     "net451": {},
     "netstandard1.3": {
-      "imports": [
-        "portable-net451+win8"
-      ]
     }
```

Comment:
Should closing braces be on the same line as the opening braces for consistency with `net451`?